packages/plugin-core/assets/dendron-ws/tutorial/tutorial.1-navigation-basics.md

```diff
@@ -0,0 +1,47 @@
+---
+id: vXh-HmSTsmgGA9j9IzcZh
+title: Navigation Basics
+desc: ''
+updated: 1624446320758
+created: 1624332288808
+---
+
+Let's do a brief overview on how to navigate the Dendron UI.
```

Comment:
If feasible, I would move this to the 2nd part. Would be nice if we let people create notes and do things before introducing them concepts

Reply:
Unfortunately there's a bit of a circular dependency - Part 2 has references to things like tree view, editor vs preview. I think I'll leave the ordering as is for now, but simplify some of the descriptions in 1-navigation-basics